### DISCUSSION TITLE
Prepare a list of resources for learning Julia

### DISCUSSION THREAD

**Discussion by jiweiqi** [DENG-MIT/Arrhenius.jl#50] [Ideas]
As Julia is not widely adopted in the Combustion community, it will be great to have instructions for beginners from different backgrounds. For example,

+ without any programming experience
+ knowing Fortran/Chemkin well
+ knowing Cantera's python/Matlab interface well

**Comment by jiweiqi** [comment]
+ [Introduction to Julia from NREL-SIIP](https://nbviewer.jupyter.org/github/nrel-siip/SIIPExamples.jl/blob/master/notebook/1_introduction/an_introduction_to_julia.ipynb)

**Reply by RSuryaNarayan** [reply]
This is a great idea. I think since we are using differential programming we can definitely include tutorials from JuMP, NLSolve, Optim.jl, DifferentialEquations.jl and ForwardDiff.jl

**Reply by jiweiqi** [reply]
I am searching for existing sources so that we don't have to make it ourselves
